bgcval2/analysis_timeseries.py

```diff
@@ -139,6 +163,180 @@ def build_list_of_suite_keys(suites, debug=True):
     return analysis_keys
 
 
+def load_function(functionname):
+    """
+    Using the named function in the key yaml, load that function and return it. 
+    """
+    # load functions:
+    if functionname in std_functions.keys():
+        print( "Standard Function Found:", functionname)
+        func = std_functions[functionname]
+
+    if functionname.find(':') > -1:
+        [functionFileName,functionname] = functionname.split(':')
+        lst = functionFileName.replace('.py','').replace('/', '.').split('.')
+        modulename =  '.'.join(lst)
+
+        print("parseFunction:\tAttempting to load the function:",functionname, "from the:",modulename)
+        mod = __import__(modulename, fromlist=[functionname,])
+        func = getattr(mod, functionname)
+    return func     
+
+
+def load_function_kwargs(dictionary, m_or_d):
+    """
+    Loads key word arguments from yaml file dictionairy.
+    """
+    data_kwargs = {}
+    #, data_convert_kwargss = [], []
+    #####
+    # Looking for kwargs to pass to convert:
+    for key, value in dictionary.items():
+        searchFor =  m_or_d+'_convert_'
```

Comment:
```suggestion
        searchFor =  m_or_d + '_convert_'
```

Reply:
also, `m_or_d` has a very cryptic name, can you maybe make it more descriptive?

Reply:
Changed.